src/auxiliary/auxiliary.jl

```diff
@@ -156,7 +148,8 @@ For example, use `solve(ode, alg; ode_default_options()...)`
 """
 function ode_default_options()
   if mpi_isparallel()
-    return (; save_everystep = false, internalnorm = ode_norm, unstable_check = ode_unstable_check)
+    return (; save_everystep = false, internalnorm = ode_norm,
+            unstable_check = ode_unstable_check)
```

Comment:
This looks a bit weird to me - I would have expected something like
```julia
    return (; save_everystep = false, internalnorm = ode_norm,
              unstable_check = ode_unstable_check)
```

Reply:
Agreed. However, I would be able to live with this.